test/ngAnimate/ngSwapSpec.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+describe("ngSwap", function() {
+
+  beforeEach(module('ngAnimate'));
+  beforeEach(module('ngAnimateMock'));
+
+  var element;
+  afterEach(inject(function($$jqLite) {
+    dealoc(element);
+  }));
+
+  it('should render a new container when the expression changes', inject(function($rootScope, $compile, $animate) {
```

Comment:
Instead of injecting `$rootScope`, `$compile`, `$animate` in every test, why not inject them in a `beforeEach` block and make them available to all tests ?